LLama/LLamaEmbedder.cs

```diff
@@ -137,8 +150,54 @@ public async Task<IReadOnlyList<float[]>> GetEmbeddings(string input, Cancellati
             embedding.EuclideanNormalization();
         }
 
-        Context.NativeHandle.KvCacheClear();
+        Context.Dispose();
 
         return (results, tokens.Length);
     }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="text"></param>
+    /// <returns></returns>
+    public int CountTokens(string text)
```

Comment:
`CountTokens` and `GetTokens` methods are duplicated on `LLamaEmbedder` and `LLamaStatelessExecutor`. Also I don't think either of them actually requires a context (which is a very expensive object to create and destroy)!

Can these methods be moved up to `LLamaWeights` class instead? That's a more appropriate place for methods relating to tokens/vocabulary.

Reply:
Martin, on a second thought, I think that you may be right (only the params need to be kept). I will look at it!